### PR TITLE
248 onboarding

### PR DESCRIPTION
**Pull request by wbaccinelli**
Part of the solution to https://github.com/PerfectFit-project/virtual-coach-issues/issues/248.

These are the implementation of the API endpoints to get and accept the connection requests sent by the users to the therapist.

### CODE REVIEW

File: niceday-api/api/openapi.yaml [hunk 1]
```diff
@@ -106,6 +114,26 @@ paths:
...
+              required:
+                - invitation_id
+              type: object
+              properties:
+                invitation_id:
+                  type: string
+                  description: ID of the connection request
+                  example: "38527"
+      responses:
+        "200":
```
Comment (raar1) [inline niceday-api/api/openapi.yaml:141]: Can this only ever be successful?

Comment (raar1) [reply niceday-api/api/openapi.yaml:141]: Looks like 500 is also a possible response (also for the connectionrequests endpoint). Not sure if it really matters whether they are in the openapi yaml or not.

**Review by raar1 — APPROVED** [review]
Looks good to me, left some minor comments

File: niceday-api/api/openapi.yaml (hunk 1 above)
Comment (wbaccinelli) [reply niceday-api/api/openapi.yaml:141]: We still get an error response from the service, but I agree that it is not the cleanest way

**Pull request merged**